### PR TITLE
Update: Support embed

### PR DESCRIPTION
**Pull request by Charliego93**

### CODE REVIEW

**Comment by thinkerou** [comment]
@whimthen please add some unit test, thanks!

**Comment by whimthen** [comment]
> @whimthen please add some unit test, thanks!

Unit test has been added.

File: i18n.go [hunk 1]
```diff
@@ -12,13 +12,17 @@ func newI18n(opts ...Option) {
 	ins := &ginI18nImpl{
 		getLngHandler: defaultGetLngHandler,
 	}
-	ins.setBundle(defaultBundleConfig)
+	//ins.setBundle(defaultBundleConfig)
```
Comment (appleboy) [inline i18n.go:15]: remove unused code.

**Review by appleboy — CHANGES_REQUESTED** [review]

**Review by Charliego93 — COMMENTED** [review]
Reviewed it.

**Comment by appleboy** [comment]
@Charliego93 Please fix the build error.

**Comment by Charliego93** [comment]
> I want this feature. Thanks!

I have no time check it for now. It's later. Maybe you can use my another package https://github.com/Charliego93/go-i18n

**Comment by shuntagami** [comment]
@Charliego93 
Thanks for your work. I'll check it later.

@appleboy
May I work on this PR instead?

**Comment by appleboy** [comment]
@shuntagami OK, Great. Go ahead. Thanks

**Comment by Charliego93** [comment]
@appleboy I just saw the build error because embed is not supported until 1.16, workflow strategy need to be modified. Can I modify it?

**Comment by appleboy** [comment]
@Charliego93 You can add a build tag (`//go:build`) to separate multiple versions. see https://kofo.dev/build-tags-in-golang

**Comment by appleboy** [comment]
@Charliego93 closed via https://github.com/gin-contrib/i18n/pull/7

**Pull request closed without merging**